samples/sample_calibrate_eye_to_hand.py

```diff
@@ -75,6 +77,14 @@ def _main():
     else:
         print("FAILED")
 
+    print("Transforming calibration result")
+    transformed_numpy_matrix = calibration_result.transform()
+
+    print("Saving transformed calibration result")
+    with tempfile.TemporaryDirectory() as tempdir:
+        file_path = os.path.join(tempdir, "hand_eye.yml")
```

Comment:
`os.path` is old-school for most things.
The modern way would be `file_path = Path(tempdir) / "hand_eye.yml"`